.gitignore

```diff
@@ -67,6 +67,21 @@ target/
 # VS Code
 .vscode/
 
+# OSX files
+.DS_Store
+
+# pytest
+.pytest_cache/
+
+# mypy
+.mypy_cache/
+
+# pyenv
+.python-version
+
+# asdf
+.tool-versions
```

Comment:
Shouldn't these be check in? To make sure everyone uses the exact same version?

Reply:
Nej

Reply:
Jag vet inte hur stödet är för asdf på exempelvis Windows-datorer, så jag tror det kan vara svårt att se till att synka detta, och jag är inte helt säker på att jag vill tvinga på alla att använda asdf, även om jag tycker det är ett bra verktyg.

Om ni tycker att vi ska ha det kan ni ta upp det separat. Tills vidare har vi det inte.

Reply:
Lite mer utvecklat är mina tankar såhär - *jag* använder `asdf` för att hantera NodeJS, och tills att någon annan vill införa det som den officiella metoden tänker jag mig att det är praktiskt att jag och andra som använder det enkelt kan komma igång.

Jag köper inte argumentet med att alla personer borde ha en egen global .gitignore, det tror jag är långt ifrån vanligt, det är bättre att minska friktion för att hjälpa till istället. Det vanliga som jag sett i de projekt jag hjälpt till i är att man lägger in tooling-filer ganska friskt i .gitignore-filer för att folk skall kunna vara effektiva och komma igång snabbt. För vad är skadan av en lång .gitignore annat än något artificiellt mål om "prydlighet"?